misk-api/src/main/kotlin/misk/client/ClientAction.kt

```diff
@@ -17,7 +17,7 @@ data class ClientAction(
   /** The return type fo the action */
   val returnType: KType
 ) {
-  internal constructor(clientName: String, method: KFunction<*>) :
```

Comment:
We need to be able to create `ClientAction`s in Armeria services for cross-framework libs.